src/widgets/DetailRow.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2016
+ */
+
+import React from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { SimpleLabel } from './SimpleLabel';
+
+/**
+ * Simple layout component render sequential SimpleLabel components
+ * in a row.
+ *
+ * Each SimpleLabel has a label and some other text.
+ *
+ * @prop {Array} details Array of objects in the form {label, text}
+ */
+export const DetailRow = ({ details }) => {
+  const Details = React.useCallback(
+    () =>
+      details.map(({ label, text }) => (
+        <View>
```

Comment:
should have a `Key` on the `View`

Reply:
e43d55f892a0181d970a74864b87b03a800c0abf